src/main/java/org/jabref/model/entry/Month.java

```diff
@@ -93,9 +97,28 @@
             int number = Integer.parseInt(value);
             return Month.getMonthByNumber(number);
         } catch (NumberFormatException e) {
-            return Optional.empty();
+            return parseGermanShortMonth(testString);
         }
     }
+    
+    /**
+	 * Parses a month having the string in German standard form such as
+	 * "Oktober" or in German short form such as "Okt"
+	 * 
+	 * @param value,
+	 *            a String that represents a month in German form
+	 * @return the corresponding month instance, empty if input is not in German
+	 *         form
+	 * @author Johannes Pre�mar
```

Comment:
Please do not put `@author` tags. We acknowledge our authors in `AUTHORS`. See https://github.com/JabRef/jabref/blob/master/CONTRIBUTING.md#author-credits. Please ensure that your git configuration `user.name` and `user.email` is correct.

Reply:
Okey I understand, but I have done that for PE1718, but the worksheet said so for authorize. I deleted it and will put my name directly in the Pull Request.